responses/05.0_trigger.md

```diff
@@ -3,6 +3,7 @@ Nice, you just added an action block to your workflow file! Here are some import
 - `jobs:` is the base component of a workflow run
 - `build:` is the identifier we're attaching to this job
 - `name:` is the name of the job, this is displayed on GitHub when the workflow is running
+- `runs-on:` is **required** for the type of machine the job [`runs-on`](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on). The machine can be either a GitHub-hosted runner or a self-hosted runner.
```

Comment:
```suggestion
- `runs-on:` defines the type of machine to run the job on. The machine can be either a GitHub-hosted runner or a self-hosted runner.
```

I think this would bring it more in line with the style of the rest of the definitions in this list. 🙂